migrate/20240320_add_vm_provision_timestamps.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:vm) do
+      add_column :allocated_at, :timestamptz, null: true, default: nil
+      add_column :provisioned_at, :timestamptz, null: true, default: nil
```

Comment:
You don't need a default with null: true right? Is this because of that sequel rubocop?

Reply:
You are right, `null: true, default: nil` are default values. I just copied from https://github.com/ubicloud/ubicloud/blob/f39710b2479ef8a1d1446d97692c7cdd90b39b9f/migrate/20240209_postgres_multiple_servers.rb#L8 😄 

I removed these values